src/main/java/com/avispl/symphony/dal/infrastructure/management/samsung/smartthings/common/AggregatorGroupControllingMetric.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+
+package com.avispl.symphony.dal.infrastructure.management.samsung.smartthings.common;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * SmartThingsManagementGroupMetric
```

Comment:
Should it be "AggregatorGroupControllingMetric" here.

Reply:
I fixed it